### PR TITLE
Update the get api

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

File: experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveTableView.java [hunk 1]
```diff
@@ -65,4 +69,14 @@ public String getBaseLocation() {
...
+
+  @Override
+  public TableViewInfo toProto(String viewName) {
+    // TODO(gpang): implement clustered columns.
+    return TableViewInfo.newBuilder()
+        .setViewName(viewName)
+        .setViewType("todo: hive parquet")
+        .addClusteredColumns(mTable.getSchema().getCols(mTable.getSchema().getColsCount() - 1))
+        .build();
+  }
```
Comment (yuzhu) [inline experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveTableView.java]: i think the information you are looking for here is stored in `mPartitionCols`, the list of columns the table is partitioned / clustered by?

Comment (gpang) [reply experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveTableView.java]: wow, I didn't realize that. Updated.

**Comment by gpang** [comment]
tests are flaky

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**